db/migrate/20111220152525_create_wiki_to_issue_refs.rb

```diff
@@ -1,4 +1,4 @@
-class CreateWikiToIssueRefs < ActiveRecord::Migration
+class CreateWikiToIssueRefs < Rails.version < '5.1' ? ActiveRecord::Migration : ActiveRecord::Migration[4.2]
```

Comment:
```suggestion
class CreateWikiToIssueRefs < (Rails.version < '5.1') ? ActiveRecord::Migration : ActiveRecord::Migration[4.2]
```
Would it work to include brackets to make it clear that it's a ternary operator?

Reply:
It makes sense. I'll do a quick check on monday and commit the suggestion. Thanks